### PR TITLE
added project context example

### PR DESCRIPTION
**Pull request by ventura-rivera**
# Citrine Python PR

## Description 
Including an example of user authorization and project context to the docs.

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### Adherence to team decisions
- [ ] I have added tests for 100% coverage
- [ ] I have written Numpy-style docstrings for every method and class.
- [x] I have communicated the downstream consequences of the PR to others.

### CODE REVIEW

**Comment by lbianchini84** [comment]
Could you add an example like Dataset A is owned by Project A and share with read access to Project B. 

User has write access to project A and B. They can not edit/delete/update data in Dataset A from project B

**Comment by ventura-rivera** [comment]
> Could you add an example like Dataset A is owned by Project A and share with read access to Project B.
> 
> User has write access to project A and B. They can not edit/delete/update data in Dataset A from project B

@lbianchini84 sure I can add that

**Comment by ventura-rivera** [comment]
> @ventura-rivera I think it makes to specify that they can have write access to dataset A and project B but still can't edit Dataset A from Project B. Does that make sense?

@lbianchini84 that makes sense. I added that further clarity.

File: docs/source/getting_started/projects_and_users.rst [hunk 1]
```diff
@@ -7,6 +7,11 @@ Overview
 
 Projects are containers for other resources, such as datasets and modules.
 Access rights on resources inside a Project are managed, granted, and revoked at the Project level.
+This means a user can have permission to perform an action in one project, but be rejected because 
+they are trying to do it from the wrong project. For example, consider Dataset A owned by Project A.
```
Comment (ahl389) [inline docs/source/getting_started/projects_and_users.rst]: `trying to do it from the wrong project` - I might use more specific language to clarify what "from" the wrong project means.

Comment (ventura-rivera) [reply docs/source/getting_started/projects_and_users.rst]: > `trying to do it from the wrong project` - I might use more specific language to clarify what "from" the wrong project means.

Ok I tried to use more helpful language. Let me know if you have any more suggestions.

**Comment by ventura-rivera** [comment]
@ahl389 Is this good to be merged yet?

**Comment by ahl389** [comment]
@ventura-rivera yea - approved

**Pull request merged**